src/Ionisation.jl

```diff
@@ -29,12 +30,24 @@ function ionrate_fun!_ADK(ionpot::Float64, threshold=true)
         thr = 0
     end
 
+    Ip_au = ionpot / au_energy
+    F0_au = (2Ip_au)^(3/2)
+    F0 = F0_au*au_Efield
+    avfac = sqrt.(3/(π*F0))
+
+
     ionrate! = let nstar=nstar, cn_sq=cn_sq, ω_p=ω_p, ω_t_prefac=ω_t_prefac, thr=thr
         function ir(E)
             if abs(E) >= thr
-                (ω_p*cn_sq*
-                (4*ω_p/(ω_t_prefac*abs(E)))^(2*nstar-1)
-                *exp(-4/3*ω_p/(ω_t_prefac*abs(E))))
+                if cycle_average
+                    (avfac*sqrt(abs(E))*ω_p*cn_sq*
+                    (4*ω_p/(ω_t_prefac*abs(E)))^(2*nstar-1)
```

Comment:
Wouldn't it be nice to factorise the common parts of the two cases? So just the part that changes is inside the conditional?